CHANGELOG.md

```diff
@@ -37,6 +37,10 @@
 
 # cronjob
 
+## 1.0.2
+
+- add args to let cronjobs do something
```

Comment:
```suggestion
- add `args` to allow passing arguments to the container's entrypoint
```
vgl. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#container-v1-core
vgl. https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#define-a-command-and-arguments-when-you-create-a-pod